File: Projeto-Angular/Angular/src/app/models/Funcionario.ts
export class Funcionario{

    /**
     *
     */
    constructor() {
        this.id = 0;
        this.nome = '';
        this.departamentoId = 0;
        this.foto = '';
        this.rg = '';
    }

    public id: number;
    public nome: string;
    public foto: string;
    public rg: string;
    public departamentoId: number;
}

File: WebAPI/Models/Funcionario.cs
using System;

namespace WebAPI.Models{
public class Funcionario{

    public int id { get; set; }

    public string nome { get; set; }

    public string foto { get; set; }

    public string RG { get; set; }    
    
    public int DepartamentoId { get; set; }
    
    public Departamento Departamento { get; set; }

    public Funcionario(){

    }

	public Funcionario(int id, string nome, string foto, string RG, int DepartamentoId)
    {
        this.id = id;
        this.nome = nome;
        this.foto = foto;
        this.RG = RG;
        this.DepartamentoId = DepartamentoId;

        }
}
}